src/SprykerEco/Yves/Adyen/Theme/default/components/molecules/adyen-credit-card/adyen-credit-card.ts

```diff
@@ -0,0 +1,74 @@
+declare var csf: any;
+
+import Component from 'ShopUi/models/component';
+import ScriptLoader from 'ShopUi/components/molecules/script-loader/script-loader';
+
+// Define a custom style.
+const stylesConfig = {
+    base: {
+        color: '#333',
+        fontSize: '14px',
+        fontSmoothing: 'antialiased',
+        fontFamily: 'Helvetica'
+    },
+    error: {
+        color: '#b2171a'
+    },
+    placeholder: {
+        color: '#d8d8d8'
+    },
+    validated: {
+        color: '#4fc2a0'
+    }
+};
+
+export default class AdyenCreditCard extends Component {
+    scriptLoader: ScriptLoader
```

Comment:
Make all properties and getter `protected`.

Reply:
fixed